app/src/main/java/com/hedvig/app/feature/offer/OfferActivity.kt

```diff
@@ -51,6 +53,23 @@ class OfferActivity : BaseActivity(R.layout.activity_offer) {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        offerRoot.useEdgeToEdge()
+        offerRoot.setEdgeToEdgeSystemUiFlags(true)
+
+        offerToolbar.doOnApplyWindowInsets { view, insets, initialState ->
+            view.updatePadding(top = initialState.paddings.top + insets.systemWindowInsetTop)
+        }
+
+        offerHeader.doOnApplyWindowInsets { view, _, initialState ->
+            view.updatePadding(top = initialState.paddings.top + offerToolbar.measuredHeight)
```

Comment:
It is to add padding so the layout dont start under the transparent toolbar. The only other way i can see is to add manual padding to `offerheader` in XML but that seems like it could cause problems on other screen sizes

Reply:
We should be able to pad the scroll content container with `?actionBarSize` and then use a windowInsetListener to increase that padding by `systemWindowInsetTop`

Reply:
I now realize that the extra padding from `offerToolbar.measuredHeight` wasn't even needed 😅 Dont know why i added it